sdk/javascript/packages/core/src/keeper.ts

```diff
@@ -687,9 +702,12 @@ export const initializeStorage = async (storage: KeyValueStorage, oneTimeToken:
     await platform.generatePrivateKey(KEY_PRIVATE_KEY, storage)
 }
 
-export const getSecrets = async (options: SecretManagerOptions, recordsFilter?: string[]): Promise<KeeperSecrets> => {
+export const getSecrets = async (options: SecretManagerOptions, recordsFilter?: string[], requestLinks?: boolean): Promise<KeeperSecrets> => {
```

Comment:
Don't modify this, users which want links will call getSecrets2 which already has this flag in options.